Mathlib/Analysis/Convex/Normed.lean

```diff
@@ -153,3 +155,56 @@ theorem isConnected_setOf_sameRay_and_ne_zero {x : E} (hx : x ≠ 0) :
   simp_rw [← exists_pos_left_iff_sameRay_and_ne_zero hx]
   exact isConnected_Ioi.image _ (continuous_id.smul continuous_const).continuousOn
 #align is_connected_set_of_same_ray_and_ne_zero isConnected_setOf_sameRay_and_ne_zero
+
+end SeminormedAddCommGroup
+
+section NormedAddCommGroup
+variable [NormedAddCommGroup E] [NormedSpace ℝ E] [FiniteDimensional ℝ E] {s : Set E} {x : E}
+
+/-- We can intercalate a polyhedron between a point and one of its neighborhoods. -/
+lemma exists_mem_interior_convexHull_finset (hs : s ∈ 𝓝 x) :
+    ∃ t : Finset E, x ∈ interior (convexHull ℝ t : Set E) ∧ convexHull ℝ t ⊆ s := by
```

Comment:
Can you add the auxiliary lemma "we can intercalate a simplex between a point and one of its neighborhoods"?

Reply:
I've started working on it but it's quite a lot harder because of missing API. See #14682